components/org.wso2.carbon.identity.application.authenticator.facebook/src/main/java/org/wso2/carbon/identity/application/authenticator/facebook/FacebookAuthenticator.java

```diff
@@ -339,24 +352,19 @@ protected void setSubject(AuthenticationContext context, Map<String, Object> jso
 
     protected void buildClaims(AuthenticationContext context, Map<String, Object> jsonObject)
             throws ApplicationAuthenticatorException {
+
         if (jsonObject != null) {
             Map<ClaimMapping, String> claims = new HashMap<>();
             String claimUri;
             Object claimValueObject;
-            String claimDialectUri = getClaimDialectURI();
-            if (claimDialectUri == null) {
-                claimDialectUri = "";
-            } else {
-                claimDialectUri += "/";
-            }
 
             for (Map.Entry<String, Object> userInfo : jsonObject.entrySet()) {
-                claimUri            = userInfo.getKey();
+                claimUri            = getEffectiveClaimUri(getClaimDialectURI(), userInfo.getKey());
```

Comment:
remove the extra space after variable name

Reply:
done